doc/api-extensions.md

```diff
@@ -2285,3 +2285,11 @@ This parameter allows bypassing the project name filter.
 ## `metadata_configuration`
 Adds the `GET /1.0/metadata/configuration` API endpoint to retrieve the generated metadata configuration in a JSON format. The JSON structure adopts the structure ```"configs" > `ENTITY` > `ENTITY_SECTION` > "keys" > [<CONFIG_OPTION_0>, <CONFIG_OPTION_1>, ...]```.
 Check the list of {doc}`configuration options </config-options>` to see which configuration options are included.
+
+## `device-initial-configuration`
+
+This API extension provides the capability to set initial configurations for instance root devices.
+Initial configurations are prefixed with `initial.` and can be specified either through profiles or directly during
+instance initialization using the `--device` flag.
+Note that these configuration are applied only at the time of instance creation and subsequent modifications have
+no effect on existing devices.
```

Comment:
```suggestion

Note that these configurations are applied only when the instance is created. Subsequent modifications have no effect on existing devices.
```

Reply:
Not sure if we need the empty line (to create a new paragraph), but I think it will look nicer and make it easier to read than one huge paragraph.

Reply:
To me it makes sense. I've added it. Thanks :)